install.sh

```diff
@@ -294,15 +317,15 @@ if [[ "${#chowns[@]}" -gt 0 ]]; then
   printf "%s\n" "${chowns[@]}"
 fi
 if [[ "${#chgrps[@]}" -gt 0 ]]; then
-  ohai "The following existing directories will have their group set to ${tty_underline}admin${tty_reset}:"
+  ohai "The following existing directories will have their group set to ${tty_underline}${GROUP}${tty_reset}:"
   printf "%s\n" "${chgrps[@]}"
 fi
 if [[ "${#mkdirs[@]}" -gt 0 ]]; then
   ohai "The following new directories will be created:"
   printf "%s\n" "${mkdirs[@]}"
 fi
 
-if should_install_command_line_tools; then
+if ! "$HOMEBREW_ON_LINUX" && should_install_command_line_tools; then
```

Comment:
Instead I'd suggest `should_install_command_line_tools` return false if `HOMEBREW_ON_LINUX` is set. This will simplify all the callers.